sections/futures/TradeCrossMargin/CrossMarginInfoBox.tsx

```diff
@@ -164,12 +178,27 @@ function MarginInfoBox({ editingLeverage }: Props) {
 							</PreviewArrow>
 						),
 					},
+					'Keeper ETH Deposit':
+						orderType === 'limit' || orderType === 'stop'
+							? {
+									value: formatCurrency('ETH', keeperEthBal) + 'Ξ',
+									valueNode: (
+										<>
+											{keeperEthBal.gt(0) && (
+												<ActionButton hideBorder onClick={() => setOpenModal('keeper-deposit')}>
```

Comment:
Should we have a similar "deposit" arrow? As I user I could do the following:
* Open first limit order, which deposits Keeper ETH
* Withdraw my ETH
* Now my orders won't execute, and I can't deposit ETH without a new order

Reply:
I've built the UI here but commented out for now until we have a deposit function on the smart contract.